core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -193,6 +215,10 @@
         </li>
       </ul>
       <hint-and-solution-buttons ng-if="windowIsNarrow"></hint-and-solution-buttons>
+      <div class="star-container" ng-if="windowIsNarrow">
+          <i class="material-icons star">star_rate</i>
```

Comment:
Indentation is wrong. Also we generally use a different style for the content of the i tags between material icons; look elsewhere in the codebase.

Reply:
Done!